bbmri/modules/eric-setup.sh

```diff
@@ -6,6 +6,13 @@ if [ "${ENABLE_ERIC}" == "true" ]; then
 
 	# Set required variables
 	ERIC_BROKER_ID=broker.bbmri.samply.de
```

Comment:
In case it's test env you set them twice, first for production, and then for test
do it in if-then-else instead